docs/tutorial.md

```diff
@@ -396,11 +396,12 @@ What's going on here? Let's step through the code.
 
 ## Extracting SLiM individuals
 
-### Extracting a sample of individuals
+Another important thing to be able to do is to extract particular
+individuals from a simulation,
+for analysis or for outputting their genotypes, for instance.
+This section demonstrates some basic manipulations of individuals.
```

Comment:
this introductory bit seemed like it applied to this whole section, not the sub-setction

Reply:
I thought the phrasing in that first sentence "particular individuals" suggested keeping track of specific individuals as opposed to taking a random sample. Maybe just delete "particular" ?